modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
```

Comment:
this one is important as not to trigger [OCPBUGS-7780](https://issues.redhat.com/browse/OCPBUGS-7780)-alike bugs. I suggest we should move this one into an 
```
Important:
```
block